.github/workflows/base.yml

```diff
@@ -48,6 +48,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      # Install pyright
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
```

Comment:
I would rather prefer not stick to conda to install everything needed. Could it be possible to rather use [conda] "install" in the nox file to install pyright (instead of npm), and to remove this added `actions/node` from the CI build ?

That way there is absolutely no difference between what runs on developer laptop and GHA (same nox session, and everything is installed and runs in the nox sessions).